core/controllers/custom_landing_pages.py

```diff
@@ -30,22 +29,26 @@ class FractionLandingRedirectPage(base.BaseHandler):
     @acl_decorators.open_access
     def get(self):
         """Handles GET requests."""
-        self.redirect('/learn/maths/fractions')
+        self.redirect('/math/fractions')
+
+
+class TopicRedirectPage(base.BaseHandler):
```

Comment:
Make this TopicLandingRedirectPage (As this can create conflict with Topic (new structure) pages)

Reply:
Done.